### DISCUSSION TITLE
Cakes

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1138] [Art]
This continues the ongoing request for cake designs. We will begin to release these weekly into the game starting on Monday 🚀 

One of the big challenges we found with the previous cake designs was their large size. We had a range of inconsistencies and the cakes looked too big inside the UI modals and when placed near the cake stall.

To fit with the current aesthetics, we will be looking for heavily pixellated designs.

**Cake Types**

Please submit art for either or all of the following:

- Potato Cake
- Pumpkin Cake
- Carrot Cake
- Cabbage Cake
- Parsnip Cake
- Beetroot Cake
- Cauliflower Cake
- Radish Cake
- Wheat Cake

**Requirements**

- Raw minified file
- Screenshot of cake on the table near the cake stall
- 12x12px dimensions

See example of a radish cake.

_Minified_

![radish_cake](https://user-images.githubusercontent.com/11745561/175441701-e7f0528c-f43f-4790-a1cf-7a2997a469b6.png)

_Preview_

![radish_cake](https://user-images.githubusercontent.com/11745561/175441673-ad2ce0e6-d36a-443b-926f-77a9a35cb26f.png)

_In game_

<img width="357" alt="Screen Shot 2022-06-24 at 11 10 52 am" src="https://user-images.githubusercontent.com/11745561/175441660-2d3055a6-b822-4000-a749-a49ec910ee04.png">

**Deadline**

Each week the cakes are a surprise so feel free to continue to submit art over the next 10 weeks. Hopefully we can get your cake into the game! ❤️

**Comment by Kiulg** [comment]
There is some order in mind?

Meanwhile this is my cake of baked potato slices xD
![Cake](https://user-images.githubusercontent.com/18270876/175451510-ef899bc9-f4e0-485e-82f4-c2d6f00c2c66.png)![Potatom](https://user-images.githubusercontent.com/18270876/175451577-5e22cb98-917f-4a8f-ae37-26fdb9537657.png)

**Comment by benaxol12** [comment]
I worked on the radish cake given as an example 😋 

![Cakes](https://user-images.githubusercontent.com/98098031/175475101-fbcc31e2-6521-42e0-98b2-570c981fa166.gif)


### **Preview:**
- Potato Cake
![PotatoCakex6](https://user-images.githubusercontent.com/98098031/175477043-2e6ff1fb-8331-4fac-82c0-9490cea1d467.png)

- Pumpkin Cake
![PumpkinCakex6](https://user-images.githubusercontent.com/98098031/175477127-83547d42-ff4c-4106-bce0-b89281959835.png)

- Carrot Cake
![CarrotCakex6](https://user-images.githubusercontent.com/98098031/175477059-038ee209-54ae-4353-8a4c-fbbfcc83da1b.png)

- Cabbage Cake
![CabbageCakex6](https://user-images.githubusercontent.com/98098031/175477067-8c984059-7eff-44f3-b477-3cc28bbf2b62.png)

- Parsnip Cake
![ParsnipCakex6](https://user-images.githubusercontent.com/98098031/175477146-30b9f899-8c89-4ca0-8d1f-47cee22701e3.png)

- Beetroot Cake
![BeetrootCakex6](https://user-images.githubusercontent.com/98098031/175477156-2c077315-6603-48b5-9465-bd0af6c10d08.png)

- Cauliflower Cake
![CauliflowerCakex6](https://user-images.githubusercontent.com/98098031/175477170-10479b7c-5514-479a-b86c-7058f1f7a01c.png)

- Wheat Cake
![WheatCakex6](https://user-images.githubusercontent.com/98098031/175477172-e8d45235-2b47-4c28-b96d-2ad320b98565.png)



### **Minified**
- Potato Cake ![PotatoCake](https://user-images.githubusercontent.com/98098031/175474813-49b7bfef-f1d2-466c-a942-efe1c46fa714.png)

- Pumpkin Cake  ![PumpkinCake](https://user-images.githubusercontent.com/98098031/175474941-02d1dbb9-af62-40c2-aabf-0e5354f61649.png)

- Carrot Cake ![CarrotCake](https://user-images.githubusercontent.com/98098031/175474967-1e6b3332-87ff-4caa-a258-0f16401849b7.png)

- Cabbage Cake ![CabbageCake](https://user-images.githubusercontent.com/98098031/175474999-bc43c9cf-a265-45e2-849c-255cce78e453.png)

- Parsnip Cake  ![ParsnipCake](https://user-images.githubusercontent.com/98098031/175475023-fda05d1b-ca30-4909-a64e-566a72029221.png)

- Cauliflower Cake ![CauliflowerCake](https://user-images.githubusercontent.com/98098031/175475041-bf997307-a0a1-4b6b-8f3f-d0330862a343.png)

- Wheat Cake ![WheatCake](https://user-images.githubusercontent.com/98098031/175475053-904f0a8f-3314-4b2b-b11c-b42da7f6341e.png)

**Comment by tworkyfree** [comment]
Cake is my favourite food)
Discord: Xlite#0092
Farm: 129009

- Potato Cake
![cake1](https://user-images.githubusercontent.com/104664729/175489740-4ecaacec-672b-4696-aa63-a7ce552f6c5d.png)
![cake1big](https://user-images.githubusercontent.com/104664729/175759953-cb8c7435-09fe-4626-88e9-e7a98c46873b.png)
![cake1_stall-export](https://user-images.githubusercontent.com/104664729/175516378-2063f88f-5fcf-465e-9280-73e47c8398b6.png)


- Pumpkin Cake 
![cake2](https://user-images.githubusercontent.com/104664729/175759977-6b531fa6-fa60-4f17-8806-3532cc68030c.png)
![cake2bigt](https://user-images.githubusercontent.com/104664729/175759981-3383955b-92ca-48c4-9453-7ea9bafdbb4c.png)
![cake2_stall-export](https://user-images.githubusercontent.com/104664729/175516710-9a744cdf-d66f-4012-a821-5c85042b22be.png)



- Carrot Cake 
![cake7](https://user-images.githubusercontent.com/104664729/175520808-5073016e-7bd7-4060-8065-de83d4f33153.png)
![cake7big](https://user-images.githubusercontent.com/104664729/175760162-10baeb96-9404-437f-b0be-2539586bfc11.png)
![cake7_stall-export](https://user-images.githubusercontent.com/104664729/175521024-710ced44-c20d-4859-83f5-c3da2838a5cc.png)




- Cabbage Cake 
![cake4](https://user-images.githubusercontent.com/104664729/175497195-2bb47e6e-355b-452f-9ced-ffcfb2a20640.png)
![cake4bigt](https://user-images.githubusercontent.com/104664729/175760006-94849ddd-285b-485b-a052-714ac2a333e3.png)
![cake4_stall-export](https://user-images.githubusercontent.com/104664729/175516969-cb33631d-5fe3-48e8-a98b-1e7cb07b247b.png)



- Parsnip Cake 
![cake5](https://user-images.githubusercontent.com/104664729/175500575-431f87f9-ddb1-4f7b-9486-2e849815624c.png)
![cake5bigt](https://user-images.githubusercontent.com/104664729/175760092-e8d768d5-9798-4e8d-8fdd-186a4731ede8.png)
![cake5_stall-export](https://user-images.githubusercontent.com/104664729/175517389-d857e3d3-0866-4a0b-a7d8-813d9e0639f8.png)




- Beetroot Cake
![cake8](https://user-images.githubusercontent.com/104664729/175759898-e04a8a31-db00-4d59-b669-771b78cfa883.png)
![cake8bigt](https://user-images.githubusercontent.com/104664729/175760159-0c0ee3ac-7679-4a9e-a1d8-bca785ca61d6.png)
![cake8_stall-export](https://user-images.githubusercontent.com/104664729/175759900-65ee4862-d43c-48d2-bfeb-9d7d0b96812f.png)


- Cauliflower Cake 
![cake6](https://user-images.githubusercontent.com/104664729/175502003-22ab25c8-fe93-4ad1-87db-70555c4ccf8b.png)
![cake6big](https://user-images.githubusercontent.com/104664729/175760104-4c417f4f-e00b-46eb-b648-6cabb98566ce.png)
![cake6_stall-export](https://user-images.githubusercontent.com/104664729/175518606-909228d4-f3c5-4bd4-b70d-50dc24d0c4a2.png)



- Wheat Cake 
![cake3](https://user-images.githubusercontent.com/104664729/175502804-665168e9-44f8-4d86-a0ce-f92a9a7acf1d.png)
![cake3big](https://user-images.githubusercontent.com/104664729/175760156-f14ca254-abc1-4821-88da-4815c5bb4133.png)
![cake3_stall-export](https://user-images.githubusercontent.com/104664729/175518312-9baa9184-3f0f-47bc-a2c9-03de08bd018a.png)

**Comment by MrSnorch** [comment]
- Potato Cake
![картофельный торт](https://user-images.githubusercontent.com/104712725/176494031-e5110559-bab8-4125-8d49-84a451dd3176.png)
![картофельный торт превью](https://user-images.githubusercontent.com/104712725/176494098-f3595b72-4484-40b5-8b12-901d90b9584b.png)

- Carrot Cake  
![морковный торт v2](https://user-images.githubusercontent.com/104712725/176494378-1c1ede41-4151-4bf0-993b-f9b67163ece1.png)
![морковный торт v2 превью](https://user-images.githubusercontent.com/104712725/176494382-6f4f2423-b2f9-4fd9-aef3-978efc485378.png)

- Pumpkin Cake
![торт тыква v3](https://user-images.githubusercontent.com/104712725/176494971-21036bbe-034a-40e3-9370-700e11ba5dc9.png)
![торт тыква v3 превью](https://user-images.githubusercontent.com/104712725/176494981-164473f6-055b-4e55-a338-920bfbd958de.png)

 - Beetroot Cake
![торт свела v2](https://user-images.githubusercontent.com/104712725/176495268-150b7865-fa04-4d43-a3eb-d1b79eac27fb.png)
![торт свела v2 превью](https://user-images.githubusercontent.com/104712725/176495283-b4713e09-948e-4162-b251-80c823ecd214.png)

- Cauliflower Cake 
![капустный торт](https://user-images.githubusercontent.com/104712725/175775556-133b88eb-9381-4841-8ae2-40e4d0e34133.png)
 ![капустный торт превью](https://user-images.githubusercontent.com/104712725/175826749-a0d409f3-5d7a-4079-ac4e-a57d515009fb.png)
 
- Parsnip Cake
![торт пастернак](https://user-images.githubusercontent.com/104712725/176493769-bb40939c-27f7-42e2-88e7-fa78009cc522.png)
![торт пастернак превью](https://user-images.githubusercontent.com/104712725/176493782-8846337f-664e-4143-9c57-73f8c4c6b43c.png)

 - Wheat Cake  
 ![пшеничный торт](https://user-images.githubusercontent.com/104712725/175915199-663229e6-31fe-4251-83ae-9b673452e703.png)
![пшеничный торт превью](https://user-images.githubusercontent.com/104712725/175916700-c3e30ea7-0266-4ce8-819b-3e8a48eb9063.png)

![стент v6](https://user-images.githubusercontent.com/104712725/176821551-3383b4f5-e685-407f-a924-e1ef465cd5d7.gif)

**Reply by azimutFTW** [reply]
nice bro

**Reply by gnom4ik91** [reply]
Крутяк))

**Comment by JcEii** [comment]
Here's my own made cakes, I enlarged the cake coz it's pretty delicious when it is big haha.

### In Game:
![Shop cake](https://user-images.githubusercontent.com/105348036/177065929-17f1f1a4-12ed-426d-8d6f-c3ae8a94ffa9.gif)



### Cake Preview (64x64):

- Potato Cake
![cake1](https://user-images.githubusercontent.com/105348036/177066062-820838ee-4dc5-4394-9992-b59294ea19f0.png)


- Pumpkin Cake
![cake2](https://user-images.githubusercontent.com/105348036/175561948-ea331865-0ba6-4390-b553-25ff84697a80.png)

- Carrot Cake
![cake3](https://user-images.githubusercontent.com/105348036/175561951-6fee5c61-891e-4e89-86c1-46c2676dff61.png)

- Cabbage Cake
![cake4](https://user-images.githubusercontent.com/105348036/175561954-cc2f5de2-3358-4207-a052-249483465ba3.png)

- Parsnip Cake
![cake5](https://user-images.githubusercontent.com/105348036/175561959-13bb76ab-4914-458f-91de-56fc1fcc76e8.png)

- Beetroot Cake
![cake6](https://user-images.githubusercontent.com/105348036/175561962-2634d680-14eb-4bde-bd24-83024ba7e290.png)

- Cauliflower Cake
![cake7](https://user-images.githubusercontent.com/105348036/175561967-c3524f06-8977-4947-ab2c-30130a93c5aa.png)

- Wheat Cake
![cake8](https://user-images.githubusercontent.com/105348036/175561971-6ce7195b-4df1-4ed2-a294-4a14e504ce86.png)


### Cake Minified (12x12):
- Potato Cake
![cake-minified-1](https://user-images.githubusercontent.com/105348036/177065863-99d624b5-2a8b-4083-b500-a59656c231a3.png)


- Pumpkin Cake
![cake-minified-2](https://user-images.githubusercontent.com/105348036/175562652-bda31b48-b49c-4ea8-9ce4-f465dc1942be.png)

- Carrot Cake
![cake-minified-3](https://user-images.githubusercontent.com/105348036/175562654-6c91debd-dae2-4cd9-bfea-b62c9b037946.png)

- Cabbage Cake
![cake-minified-4](https://user-images.githubusercontent.com/105348036/175562658-c0b144b3-bb09-4cb7-bc52-db2a51e8c547.png)

- Parsnip Cake
![cake-minified-5](https://user-images.githubusercontent.com/105348036/175562660-e042427b-8185-4427-b33f-b8a35e1d6008.png)

- Beetroot Cake
![cake-minified-6](https://user-images.githubusercontent.com/105348036/175562663-0a3eab6e-6ed6-402a-b92f-0ee43fcfba68.png)

- Cauliflower Cake
![cake-minified-7](https://user-images.githubusercontent.com/105348036/175562666-5e15afe8-ae46-4e86-9561-5ac487650812.png)

- Wheat Cake
![cake-minified-8](https://user-images.githubusercontent.com/105348036/175562668-7cab8564-52ea-4860-ad2d-4a8c2c307064.png)

**Comment by FarmerKuenster** [comment]
sunflower cake idea!
![sunflower-12-12cake](https://user-images.githubusercontent.com/104322749/175823764-45292d4d-7b04-442d-971d-f5e12521b396.png)
![sunflower-12-12cake-large](https://user-images.githubusercontent.com/104322749/175823802-153c1b07-8bd6-4294-a16a-ecde3fd7b220.png)


discord: farmerkunester #1626
farm id: 32391

**Comment by esp123** [comment]
![cakes-sunf1](https://user-images.githubusercontent.com/74337946/177068295-d630223e-2a84-46dc-a7b8-7b14634a5aab.png)
![cakes-sunf](https://user-images.githubusercontent.com/74337946/177068299-720fe8dc-1859-4411-9820-2aaa834f5b61.png)
sunflower cake

![cakes-potat1](https://user-images.githubusercontent.com/74337946/177088775-b12f4c9b-1a78-49d7-b35e-c0cdf853ddea.png)
![cakes-potato](https://user-images.githubusercontent.com/74337946/177088783-d4635713-21a3-4b7f-a0bd-a4a5772a4c36.png)
potato cake

![cakes-cab1](https://user-images.githubusercontent.com/74337946/177091859-8fa702cc-624e-4f19-b805-4e09ac08fd51.png)
![cakes-cabbage](https://user-images.githubusercontent.com/74337946/177091873-6b1e0091-8689-48f6-9863-e84c7c1a063a.png)
cabbage cake

**Reply by JcEii** [reply]
The potato cakeee, so cuteee <3

**Comment by sfwhat** [comment]
#### I realized there is no deadline and I draw remaining 4 cakes

Sunflower Cake
![Sprite-0003](https://user-images.githubusercontent.com/95937808/182578898-d1b8c0ff-e80b-4932-a470-f0d5f5cf7a00.png)
![sunflowercake](https://user-images.githubusercontent.com/95937808/182578893-18c4ce91-8371-496a-b9db-a2b7511a500a.png)

Beetroot Cake
![Sprite-0004](https://user-images.githubusercontent.com/95937808/182580089-572ad882-881a-4f70-84af-42cb6f6cb79f.png)
![beetrootcake](https://user-images.githubusercontent.com/95937808/182580075-098db8fc-050c-4913-bbfe-b65b8fd5452e.png)

Cauliflower Cake
![Sprite-0002](https://user-images.githubusercontent.com/95937808/182579049-47b21507-6960-42d6-91c5-7a374f5960d0.png)
![cauliflowercake](https://user-images.githubusercontent.com/95937808/182579061-036eaf1b-abc1-4697-b3ef-f733ed473c1e.png)

Parsnip Cake
![Sprite-0001](https://user-images.githubusercontent.com/95937808/182579136-3257f00e-dda7-48f6-87b3-35d88edb4dce.png)
![parsnipcake](https://user-images.githubusercontent.com/95937808/182579151-dd22b2c6-80e0-48d9-9ef5-3292c6cc1671.png)

![cakes](https://user-images.githubusercontent.com/95937808/182580128-95abf3e6-1e30-4cea-9cdc-eed810bdb321.gif)


---
SfWhat
Farm#6127
Twitter@strugwhat
